lab_3_generate_by_ngrams/main.py

```diff
@@ -404,6 +676,14 @@ def _get_next_token(
 
         In case of corrupt input arguments return None.
         """
+        if not isinstance(sequence_to_continue, tuple) or not sequence_to_continue:
+            return None
+
+        next_token = self.beam_searcher.get_next_token(sequence_to_continue)
+        if next_token is None:
```

Comment:
do we need this check? if it is none, we return this value... if it is not, we also return this value...

Reply:
fixed